utils/verifyUser.js

```diff
@@ -0,0 +1,14 @@
+const dotenv = require('dotenv');
+const jwt = require('jsonwebtoken');
+
+dotenv.config();
+const verifyUser = token => {
+  if (!token) return;
+  try {
+    const decoded = jwt.verify(token, process.env.SECRET);
+    return decoded;
+  } catch (err) {
+    throw new Error('Incorrect Token', err);
```

Comment:
It should be `Unathorized` error with `403` status code sent to client

Reply:
fixed